shared/src/dispatchers/ses/sendBulkTemplatedEmail.js

```diff
@@ -44,12 +46,57 @@ exports.sendBulkTemplatedEmail = async ({
       Template: templateName,
     };
 
-    applicationContext.logger.info('Bulk Email Params', params);
+    await exports.sendWithRetry({ applicationContext, params });
+  } catch (err) {
+    applicationContext.logger.error(`Error sending email: ${err}`, err);
+  }
+};
+
+/**
+ * Sends the email via SES with a MAX_SES_RETRIES = 10;
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext application context
+ * @param {object} providers.params the parameters to send to SES
+ * @param {number} providers.retryCount the number of retries attempted
+ */
+exports.sendWithRetry = async ({
+  applicationContext,
+  params,
+  retryCount = 0,
+}) => {
+  const SES = applicationContext.getEmailClient();
+
+  applicationContext.logger.info('Bulk Email Params', params);
+  const response = await SES.sendBulkTemplatedEmail(params).promise();
+  applicationContext.logger.info('Bulk Email Response', response);
 
-    const response = await SES.sendBulkTemplatedEmail(params).promise();
+  // parse response from AWS
+  const needToRetry = response.Status.map((attempt, index) => {
+    // AWS returns 'Success' and helpful identifier upon successful send
+    return attempt.Status !== 'Success' ? params.Destinations[index] : false;
+  }).filter(row => !!row);
```

Comment:
not even a _nit_ but an observation I made in a recent commit -- you can use `.filter(Boolean)` which gives you only the truthy entries, which is kind of nice.  no change necessary.